frontend/src/components/Pagination.cy.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import Pagination from './Pagination'
+
+describe('<Pagination />', () => {
+  let defaultProps
+  beforeEach(() => {
+    defaultProps = {
+      onClickPrev: cy.stub(),
+      onClickNext: cy.stub(),
+      onChange: cy.stub()
+    }
+  })
+  it('Desativa o botão de paginação Anterior quando na primeira página', () => {
+    // see: https://on.cypress.io/mounting-react
+    const pageInfo = {
+      "currentPage": "1",
+      "totalPages": 100,
+      "totalCustomers": 200
+    }
+    cy.mount(<Pagination currentPage={parseInt(pageInfo.currentPage)} paginationInfo={pageInfo} defaultProps />)
+    cy.contains('Prev').should('be.disabled')
+
+  })
+  it('Desativa o botão de paginação Próximo quando na última página', () => {
+    const pageInfo = {
+      "currentPage": "100",
+      "totalPages": 100,
+      "totalCustomers": 200
+    }
+    cy.mount(<Pagination currentPage={parseInt(pageInfo.currentPage)} paginationInfo={pageInfo} defaultProps />)
+    cy.contains('Next').should('be.disabled')
+
+  })
+  it('Desativa ambos os botões de paginação Anterior e Próximo quando há apenas uma página', () => {
+    const pageInfo = {
+      "currentPage": "1",
+      "totalPages": 1,
+      "totalCustomers": 200
+    }
+    cy.mount(<Pagination currentPage={parseInt(pageInfo.currentPage)} paginationInfo={pageInfo} defaultProps />)
+    cy.contains('Prev').should('be.disabled')
+    cy.contains('Next').should('be.disabled')
+  })
```

Comment:
```suggestion
  })
  
```

Entre _test cases_ vale a pena deixar uma linha em branco. Isso facilita a legibilidade.